crates/llama-cpp-bindings/build.rs

```diff
@@ -10,8 +10,8 @@ fn main() {
         "Please init submodules with `git submodule update --init --recursive` and try again"
     );
 
-    println!("cargo:rerun-if-changed=cc/*.h");
-    println!("cargo:rerun-if-changed=cc/*.cc");
+    println!("cargo:rerun-if-changed=include/engine.h");
+    println!("cargo:rerun-if-changed=src/engine.cc");
```

Comment:
There's no `cc` directory, so the config might not take effect.

For this change, I just followed [the demo](https://github.com/dtolnay/cxx/blob/master/demo/build.rs) from cxx:
```
fn main() {
    cxx_build::bridge("src/main.rs")
        .file("src/blobstore.cc")
        .flag_if_supported("-std=c++14")
        .compile("cxxbridge-demo");

    println!("cargo:rerun-if-changed=src/main.rs");
    println!("cargo:rerun-if-changed=src/blobstore.cc");
    println!("cargo:rerun-if-changed=include/blobstore.h");
}
```